seed/energy/meter_data_processor/kairos_insert.py

```diff
@@ -9,6 +9,7 @@
 from django.conf import settings
 
 _log = logging.getLogger(__name__)
+KairosDB_Batch_Insert_Size = 1000
```

Comment:
Sure, I will move the setting to config file. I just pick a number 1000, I
encountered celery worker segment fault when the inserted data is too large.

On Tue, Mar 29, 2016 at 12:50 PM, Nicholas Long <notifications@github.com>
wrote:

> In seed/energy/meter_data_processor/kairos_insert.py
> <https://github.com/SEED-platform/seed/pull/863#discussion_r57757782>:
>
> > @@ -9,6 +9,7 @@
> >  from django.conf import settings
> >
> >  _log = logging.getLogger(__name__)
> > +KairosDB_Batch_Insert_Size = 1000
>
> can you make this a config variable with 1000 as the default? Have you
> done testing on changing this number?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/SEED-platform/seed/pull/863/files/07fcd5c43cfd1d50ca513fd8bd38d7410b999749#r57757782>
>